packages/core/admin/webpack.config.js

```diff
@@ -91,7 +91,8 @@ module.exports = ({
       rules: [
         {
           test: /\.m?js$/,
-          exclude: /node_modules/,
+          // TODO remove when plugins are built separately
+          exclude: /node_modules\/(?!(@strapi\/plugin-content-manager|@strapi\/plugin-content-type-builder|@strapi\/plugin-upload)\/).*/,
```

Comment:
@alexandrebodin this is temporary until we build the plugins separately, if you now a better regex I'm in.
Currently this is what we prevent us to do some lazy loading with the not required plugins, the strapi-plugin-* (from the community) and the local ones.

Reply:
I think like this should be ignored automatically by webpack if the package has the right properties in its package.json 🤔

Reply:
that's good enough for a temp regex